docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,100 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. Based on requrested partition count, number of non polling static partitions are created to scan table portitions in parallel. Additional single partition is poller partition which scans for new data incrementally. Assumption is that there is an ordered column using which range queries can be formed. Only newly added data will be fetched by the polling. Also out of order insertion/deletion won't be supported.
```

Comment:
User might be new to terms like ```partitions```, ```non-polling``` etc. Can we introduce the user before using these?
May be use ```operator instances``` instead of ```partitions```. Also explain what polling and non-polling means.
Also explain ```ordered column```
Keep this restricted to the objective. Move the ```not-supported```mentions to another section like ```Limitations```.